packages/terra-site/src/Index.jsx

```diff
@@ -55,6 +57,14 @@ import SlidePanelTestRoutes from 'terra-slide-panel/tests/nightwatch/SlidePanelT
 import StatusTestRoutes from 'terra-status/tests/nightwatch/StatusTestRoutes';
 import TableTestRoutes from 'terra-table/tests/nightwatch/TableTestRoutes';
 import TimeInputTestRoutes from 'terra-time-input/tests/nightwatch/TimeInputTestRoutes';
+
+// Remove eslint-disable as these pacakges are published
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Based on our discussion today, we'll update these to dependencies. We also need to add doc on the case for when these should be devDeps. (Code is in master but not released and terra-site gets released)